src/cascadia/TerminalApp/IF7Listener.idl

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace TerminalApp
+{
+    [uuid("339e1a87-5315-4da6-96f0-565549b6472b")]
+    interface IF7Listener

```

Comment:
Mind adding a comment here stating what kind of classes would need to implement IF7Listener in the future? For now it's just TermControl and AppLogic, but I assume as we add more custom controls, we may run into this problem in the future.

Reply:
This is very specific to something that _actually wants to do something with f7_; i'm going to leave a comment off. somebody should ask before using it randomly ;P